docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -113,7 +117,10 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
         (warehouse, location),
         (warehouse, size),
         (warehouse))`.
- 
+    
+    `GROUP BY GROUPING SETS(GROUPING SETS((warehouse), (warehouse, product)))` is equivalent to 
```

Comment:
let's use 2 nested GROUPING SETS in the example
```
GROUPING SETS(GROUPING SETS(warehouse), GROUPING SETS((warehouse, product)))
```

Reply:
DONE